src/output.css

```diff
@@ -556,15 +556,72 @@ video {
   --tw-contain-style:  ;
 }
 
-.font-outfit{
-  font-family: Outfit, sans-serif;
+.absolute{
```

Comment:
This file has a lot of styles that already exist in TailwindCSS which is the recommended styling technology for this exercise. some of the classes you created to style have their corresponding Tailwind utility classes, so it's not necessary to change them.

Reply:
We did not edit the output file. The tailwind builder added the classes.

Reply:
I wasn't familiar with how tailwind builds utility classes under the hood when it's set up with CLI. It's very different when you use vite, but this approach is also fine.